js/src/components/contentAnalysis/ReadabilityAnalysis.js

```diff
@@ -38,6 +39,7 @@ const StyledHelpLink = styled( HelpLink )`
 class ReadabilityAnalysis extends React.Component {
 	render() {
 		const score = getIndicatorForScore( this.props.overallScore );
+		const beta = localizedData.recalibrationBetaActive;
```

Comment:
Please rename to `isRecalibrationBetaActive` or something similar to make the name more descriptive